### PR TITLE
Update installation instructions for Elastic Agent

### LINKED ISSUE #522: Update existing Fleet docs for Fleet Server
We have several docs pages that need to be updated to include Fleet Server:
- https://www.elastic.co/guide/en/fleet/current/fleet-quick-start.html
- https://www.elastic.co/guide/en/fleet/current/fleet-quick-start-traces.html
- https://www.elastic.co/guide/en/fleet/current/elastic-agent-installation.html

We should also remove the enable central management step since its removed in 7.13. We also need to update the prerequisites since Kibana security is no longer required.

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #522

Remaining todos:

- [x] Test the steps
- [x] Update screen captures

### CODE REVIEW

**Review by bmorelli25 — COMMENTED** [review]
I'll let Eric have the final say, but from my perspective, this looks great.

File: docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc [hunk 1]
```diff
@@ -1,31 +1,65 @@
...
 
 beta[]
 
-Install an {agent} on each host that you want to monitor. These steps assume
-that you're running a fresh installation. If {agent} is already running on your
-system and you want to upgrade to a new version, see <<upgrade-elastic-agent>>.
+To collect and ship data to the {stack}, install an {agent} on each host you
+want to monitor. These steps assume that you're running a fresh installation. If
+{agent} is already running on your system and you want to upgrade to a new
+version, see <<upgrade-elastic-agent>>.
```
Comment (EricDavisX) [inline docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc:10]: I love this linking so much - thank you for the effort to improve the navigability

File: docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc [hunk 2]
```diff
@@ -36,18 +70,19 @@ include::{tab-widgets}/download-widget.asciidoc[]
...
-{fleet} enrollment key? Read the
-{fleet-guide}/fleet-quick-start.html[Quick start guide] to learn how to get one
-from {fleet}.
+. From the agent directory, run the following commands to install {agent},
+enroll it in {fleet}, and start it as a service.
++
+NOTE: On macOS, Linux (tar package), and Windows, run the `install` command to
+install {agent} as a managed service, enroll it in {fleet}, and start the
+service. The DEB and RPM packages include a service unit for Linux systems with
+systemd, so use the `enroll` command instead of `install`.
```
Comment (EricDavisX) [inline docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc:79]: In a coming PR, with the help of Ruflin and team, we need to document the Docker usage... unless we want to keep that primarily in the Beats / Fleet-Server repo (just for now, due to time).  We know it is used with Kubernetes and on its own and there are significant usage options to document.  A separate topic is likely appropriate regardless

Comment (dedemorton) [reply docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc:79]: Yup, that work is being tracked in https://github.com/elastic/observability-docs/issues/622

Comment (dedemorton) [reply docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc:79]: Also noting that we just merged a PR that hooked up the docs about running on K8s.

**Review by EricDavisX — APPROVED** [review]
I think this is great.  Thanks

**Pull request merged**